### DISCUSSION TITLE
Add-on not showing in store

### DISCUSSION THREAD

**Discussion by lordratner** [hassio-addons/addon-zwavejs2mqtt#363] [General]
Hi there,

I cant seem to find the add on to install. Only the Z-wave JS add-on is available.

How can I get the Z-waveJS2MQTT add-on installed?

Thanks

**Comment by frenck** [comment]
There are some cases known where the community add-ons store gets accidentally removed.
You can re-add it using this my button

[![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://my.home-assistant.io/badges/supervisor_add_addon_repository.svg)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository)

**Reply by lordratner** [reply]
Ok that did it, thanks.

Oddly it was removed again after a restart. But it comes back when I re-add the repository. Thanks!

**Comment by cbhiii** [comment]
This was my issue -- wow! All of these updates were waiting out here? I updated to 2022.5 core and saw the breaking change afterwards about Z-wave. Had to roll back and figure out what I was missing. I was stuck at v0.37 since HA didn't tell me there was an update out there.

**_How many other people had their add-on store (and its updates available) disappear?
There should almost be a public blurb about it._**

**Reply by frenck** [reply]
There is an issue open for it @ the Suoervisor rspository. This is not an addon issue.

**Comment by GizmoGoody** [comment]
I'm adding Z-Wave to my HA setup for the first time and fortunately stumbled across this discussion. I thought I was losing my mind since "Z-Wave JS to MQTT" was not showing up under Add-Ons or Integrations. I recommend you post the repo link in the [README](https://github.com/hassio-addons/addon-zwavejs2mqtt#readme) and [DOCS](https://github.com/hassio-addons/addon-zwavejs2mqtt/blob/main/zwavejs2mqtt/DOCS.md) until the issue is fixed.

**Reply by frenck** [reply]
Sorry, I don't document issues; issues should be fixed; not documented.
